Assets/Creatures/api/IPlayable.cs

```diff
@@ -38,31 +38,31 @@ public interface IPlayable : IStateNotifier
         // Basic Attack (aka LMB):
         // - Should be unaffected by Silence
         // - Only gated by GCD (aka no specific ability cooldown)
-        AbilityExecutionResult BasicAttack(BasicAttackEvent basicAttackEvent, Vector2 mousePosition);
+        AbilityExecutionResult BasicAttack(BasicAttackEvent basicAttackEvent, Vector3 mousePosition);
```

Comment:
I had considered that, or even just baking context into the event and then I screamed 😱  and realized that would pollute the state in that event across multiple button presses since it's a singleton.

Feel like we can add a context if/when the params start growing out of control (or if mouse position isn't relevant for everything)